string name
geometry_msgs/Pose pose
geometry_msgs/Vector3 dimensions
sensor_msgs/PointCloud2 point_cloud